web/Services/AdminService.cs

```diff
@@ -147,4 +149,28 @@ public UserDbModel GetUserFromId(int userId)
         UserDbModel userFromId = _dbContext.Users.SingleOrDefault(u => u.Id == userId)!;
         return userFromId;
     }
+
+    public async Task<List<TenantDbModel>> GetTenantDbModelsFromFilter(bool[] filterArray)
+    {
+        var tenantQuery = from tenants in _dbContext.Tenants select tenants;
+        for (var currentFilter = 0; currentFilter < filterArray.Length; currentFilter++)
+        {
+            if (filterArray[currentFilter])
+            {
+                tenantQuery = currentFilter switch
+                {
+                    0 => tenantQuery.Where(t => t.Type == "Single"),
+                    1 => tenantQuery.Where(t => t.Type == "Couple"),
+                    2 => tenantQuery.Where(t => t.Type == "Family"),
+                    3 => tenantQuery.Where(t => t.HasPet == true),
+                    4 => tenantQuery.Where(t => t.ETT == true),
+                    5 => tenantQuery.Where(t => t.UniversalCredit == true),
+                    6 => tenantQuery.Where(t => t.HousingBenefits == true),
+                    7 => tenantQuery.Where(t => t.Over35 == true),

```

Comment:
Could turn this into an object with a nullable bool for each of these options. Slightly less succinct, but a bit simpler to use/understand